sources/Android/android-communications/src/androidTest/java/com/androidcommunications/polar/InstrumentedTests.kt

```diff
@@ -1,8 +1,8 @@
 package com.androidcommunications.polar
```

Comment:
Work in progress to add local unit tests. Instrumented tests are difficult to execute for library module.